examples/multiplayer-netfox-state-machine/scripts/time-display.gd

```diff
@@ -0,0 +1,21 @@
+extends Label
```

Comment:
Note to self: We could rename this script to `simple-time-display.gd` and make it shared. 

If you feel like doing it in this PR it's appreciated, but we can keep it for later as a refactor task too.

Reply:
Done